docs/diagnostics/PX1031.md

```diff
@@ -8,7 +8,8 @@ This document describes the PX1031 diagnostic.
 | PX1031 | DACs cannot contain instance methods. | Error | Unavailable | 
 
 ## Diagnostic Description
-DACs and DAC extensions cannot contain instance methods. DACs and DAC extensions are the classes that are used to read and write data. Therefore, for appropriate program architecture and design, such classes cannot contain any application logic. 
+DACs and DAC extensions cannot contain instance methods. DACs and DAC extensions are the classes that are used to read and write data. Therefore, for appropriate program architecture and design, such classes cannot contain any application logic.
+The only exception to this rule is an override of the `ToString` method which can be useful to provide DAC text representation.
```

Comment:
In theory, [`Deconstruct`](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/deconstruct#user-defined-types) is another valid case of having an instance method in DAC, but I don't expect it to be very popular (and it can also be done via a static extension method outside of DAC class)

Reply:
Let's add it if there actually is a request for it